UnitWorld_server/server/networking/ClientsGameSender.cpp

```diff
@@ -0,0 +1,7 @@
+#include "ClientsGameSender.h"
+
+using namespace uw;
+
+const unsigned int ClientsGameSender::MINIMAL_COMPLETE_STATE_PER_MINUTE(20);
+
+const xg::Guid ClientsGameSender::PLAYER_INPUT_GAME_MANAGER_CALLBACK_ID("c2ea5fc7-67a5-46b7-82d1-b3b874b9a121");
```

Comment:
Not sure what's the purpose of this; anyways, this *looks* like something you don't want to tell the world, i.e. smth you might want to move to a config file. If that's the case, then this should be addressed

Reply:
Well exactly, I don't want to tell the world, why would I by putting it in the config.json?

Here this is used to register and unregister a callback on a certain event triggered by the game manager, i.e. UserActedOnGame or something.

Reply:
Good point. So it has to stay unchanged event through different executions of the server, or event through different versions of it?

Reply:
It has to not change throughout the execution of the server, but I will surely never change this value, why would I?

This is an implementation detail, and it's possible for it not to change, so I won't change it, as simple as that.

Reply:
What I don't understand is why don't we simply create a new `Guid` each time we run the app instead of hardcoding its internal value?

Reply:
Because it would mean additional CPU time for no benefit.
Moreover the code is easier to debug when the same GUID is used since you know what GUID should be in what callbacks.

Reply:
k then